tests/conftest.py

```diff
@@ -191,6 +191,19 @@ async def guard_func(*args, **kwargs):
 util.get_local_ip = lambda: "127.0.0.1"
 
 
+@pytest.fixture(autouse=True, scope="module")
+def garbage_collection():
+    """Make sure garbage collection is run.
+
+    pytest-aiohttp did this in same test as allocation
+    this is to mimic the behavior of pytest-aiohttp, and is
+    required to avoid warnings from spilling over into next
+    test case. We run it per module and let each module track
+    handle it's own cleanup if needed.
+    """
```

Comment:
Can we adjust the docstring to better explain what's happening and improve the language?

Reply:
Tried to reword it a bit. It was rather hard to read before :)